app/config/zerodev.ts

```diff
@@ -1,10 +1,19 @@
-import { sepolia } from "viem/chains";
+import { sepolia, baseSepolia } from "viem/chains";
+
+// export const BUNDLER_URL =
+//   "https://rpc.zerodev.app/api/v2/bundler/cf030020-fa8b-45a1-a8be-b5442f37e6f3";
+// export const PAYMASTER_URL =
+//   "https://rpc.zerodev.app/api/v2/paymaster/cf030020-fa8b-45a1-a8be-b5442f37e6f3";
+// export const PASSKEY_SERVER_URL =
+//   "https://passkeys.zerodev.app/api/v3/cf030020-fa8b-45a1-a8be-b5442f37e6f3";
+
+// export const CHAIN = sepolia;
+
+export const CHAIN = baseSepolia;
 
 export const BUNDLER_URL =
-  "https://rpc.zerodev.app/api/v2/bundler/cf030020-fa8b-45a1-a8be-b5442f37e6f3";
+  "https://rpc.zerodev.app/api/v3/652c9292-be7c-4941-a62d-74716508065a/chain/84532";
```

Comment:
api key in source control - should revoke these and replace with env vars

Reply:
Fixed!